collections/README.adoc

```diff
@@ -1,3 +1,51 @@
 = Collections good practices
 
-NOTE: Work in Progress...
+Note: Unreviewed work. Please contribute to the discussion in the Automation Red Hat COP
+
+=== Collection Structure
+[%collapsible]
+====
+Explanations:: Collections should be comprised of roles collected either at the type or landscape level. See https://github.com/redhat-cop/automation-good-practices/blob/main/structures/README.adoc[the structures definition]
+
+Rationale:: Gathering and publishing collections, rather than individual roles, allows for easier distribution and particularly becomes more important when we discuss Execution Environments.
+
+====
+=== Importing Collections
+[%collapsible]
+====
+Explanations:: Collections should always be imported using requirements files within the role
+
+Rationale:: Prior art, https://www.jeffgeerling.com/blog/2020/ansible-best-practices-using-project-local-collections-and-roles
+====
+
+=== Collection Variables
+[%collapsible]
+====
+Explanations:: Often, variables will want to be defined on a collection level, but this can cause issues with roles being able to be reused.  By creating role specific and collection wide variables, this can be made clear and roles can remain reusable.
```

Comment:
```suggestion
Explanations:: Often, variables will want to be defined on a collection level, but this can cause issues with roles being able to be reused.
By defining collection wide variables and referencing them in roles' defaults variables, this can be made clear and roles can remain reusable.
Collection variables are nowhere defined explicitly and are to be documented in the collection's documentation.
```
We follow https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line